releasenotes/notes/decouple-admission-controller-webhooks-e75f98372d5edc01.yaml

```diff
@@ -1,26 +0,0 @@
-# Each section from every release note are combined when the
-# CHANGELOG.rst is rendered. So the text needs to be worded so that
-# it does not depend on any information only available in another
-# section. This may mean repeating some details, but each section
-# must be readable independently of the other.
-#
-# Each section note must be formatted as reStructuredText.
----
-upgrade:
```

Comment:
@davidor Just to confirm, there is no expected change as a result of this change series?

Reply:
I think we should delete this changelog entry :+1: